### DISCUSSION TITLE
Chatbot whatsapp

### DISCUSSION THREAD

**Discussion by Brazziltec** [chatwoot/chatwoot#2501] [Feature Requests]
Olá pessoal eu quero saber se este sistema tem chatbot que funcione no whatsapp?

**Comment by sojan-official** [comment]
We have a Dialogflow integration that can be leveraged to create a chatbot. You can also use our agent bot APIs to add a custom provider.

**Comment by Brazziltec** [comment]
iyou know Tell me someone who has this script updated that can do this service for me because I'm still a layman, I'm waiting for an answer thank you

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by pauloricardotech** [comment]
I made our chatbot using an opensource software called n8n, I integrated it with chatwoot, it's working fine...

**Reply by emanuelperpetuo** [reply]
Good afternoon, do you have any example of this that you can send?
